ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -83,6 +68,13 @@ std::unique_ptr<Process> createHTProcess(
     DBUG("Use \'%s\' as fluid_reference_density parameter.",
          fluid_reference_density.name.c_str());
 
+    // Parameter for the density of the solid.
+    auto& density_solid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__HT__density_solid}
+        "density_solid", parameters, 1);
+    DBUG("Use \'%s\' as density_solid parameter.", density_solid.name.c_str());
+
```

Comment:
I used 'Coupled groundwater flow and transport: 2. Thermohaline and 3D convection systems' form Diersch/Kolditz as a reference for the implementation. There I could not find solid line thermal expansion and the biot constant. Please, could you give e reference to a paper covering the parameters?